kickoff/jsonexport.py

```diff
@@ -10,6 +10,15 @@
 from mozilla.release.l10n import parsePlainL10nChangesets
 
 
+def generateJSONFileList():
+    links = []
+    for rule in app.url_map.iter_rules():
+        url = str(rule)
+        if url.endswith(".json"):
+            links.append((url.replace("/json/", ""), url))
```

Comment:
url.replace() looks fragile to me. Can you replace it with os.path.basename(url)? Also I'd reverse the order in the tuple (url, filename) so you it looks LTR in the template.